harness/determined/cli/_util.py

```diff
@@ -1,67 +1,62 @@
-from enum import Enum
-from typing import Any, Callable, Dict, List, Optional
+import argparse
+from typing import Dict, List
 
-from determined.common import api
-from determined.common.declarative_argparse import Arg
```

Comment:
nitpick: we import `declarative_argparse` members like this everywhere in cli.

Reply:
The cli is very old code, and it was written before we adopted the [google style guide for python imports](https://google.github.io/styleguide/pyguide.html#22-imports), which is observed in most all of the rest of the harness code.

New code in the CLI has typically followed the existing style rather than the new standard, but that's a bug not a feature IMO.

Reply:
do you mean we've banned python imports of functions/classes/constants from modules, and we only can import whole modules?

Reply:
Yes, a long time ago.